queries/pod/highlights.scm

```diff
@@ -0,0 +1,40 @@
+; A highlight file for nvim-treesitter to use
+
+[(pod_directive)
+ (head_directive)
+ (over_directive)
+ (item_directive)
+ (back_directive)
+ (encoding_directive)
+ (cut_directive)] @keyword
+
+(head_paragraph (content) @string)
+
+(over_paragraph (content) @string)
+(item_paragraph (content) @string)
+(encoding_paragraph (content) @string)
+
+(verbatim_paragraph (content) @verbatim)
+
+(interior_sequence) @variable
```

Comment:
```suggestion
(interior_sequence
  (sequence_letter) @character
  (content) @text)
```
(maybe)

Reply:
I left it alone for now b/c it didn't play too nicely with the other interiror seqs below it